packages/@lwc/ssr-compiler/src/compile-js/index.ts

```diff
@@ -205,6 +213,32 @@ const visitors: Visitors = {
     },
 };
 
+function validateDecorators(decorators: EsDecorator[]) {
+    if (decorators.length < 2) {
+        return;
+    }
+
+    const expressions = decorators.map(({ expression }) => expression);
+
+    const hasWire = expressions.some(
+        (expr) => is.callExpression(expr) && is.identifier(expr.callee, { name: 'wire' })
+    );
+
+    const hasApi = expressions.some((expr) => is.identifier(expr, { name: 'api' }));
+
+    if (hasWire && hasApi) {
+        throw generateError(DecoratorErrors.CONFLICT_WITH_ANOTHER_DECORATOR, 'api');
+    }
+
+    const hasTrack = decorators.some(({ expression }) =>
+        is.identifier(expression, { name: 'track' })
+    );
+
+    if (hasWire && hasTrack) {
```

Comment:
```suggestion
    if ((hasWire || hasApi) && hasTrack) {
```

Reply:
If we don't have tests for `@api` + `@track`, we should probably add some.

Reply:
That error message is specific to wire + other: "@wire method or property cannot be used with @{0}".

I'll add a test for that and see what the actual error message is for api + track.